api/python/cellxgene_census/src/cellxgene_census/experimental/ml/__init__.py

```diff
@@ -1,9 +1,13 @@
 """An API to facilitate use of PyTorch ML training with data from the CZI Science CELLxGENE Census."""
 
+from .encoders import BatchEncoder, DefaultEncoder, Encoder
 from .pytorch import ExperimentDataPipe, Stats, experiment_dataloader
 
 __all__ = [
     "Stats",
     "ExperimentDataPipe",
     "experiment_dataloader",
+    "Encoder",
+    "DefaultEncoder",
+    "BatchEncoder",
```

Comment:
Should these get visible in the rendered docs? I think they need to be added to `docs/python-api.rst`

Reply:
Added.